test/e2e/framework/kubelet/stats.go

```diff
@@ -246,83 +246,6 @@ func removeUint64Ptr(ptr *uint64) uint64 {
 	return *ptr
 }
 
-// GetOneTimeResourceUsageOnNode queries the node's /stats/summary endpoint
```

Comment:
We need to remove `removeUint64Ptr()` also for passing the following error:
```
Errors from staticcheck:
test/e2e/framework/kubelet/stats.go:242:6: func removeUint64Ptr is unused (U1000)
```
pull-kubernetes-verify job tends to be stable, that means we need to check the reason and log before running `/retest` as https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/84640/pull-kubernetes-verify/1190253504040538112

Reply:
Sorry about that, as `/retest` tagged on the phone and  I didn't read the log carefully.🤣